Mathlib/Algebra/Homology/ShortComplex/ModuleCat.lean

```diff
@@ -106,7 +106,7 @@ abbrev moduleCatHomologyπ : ModuleCat.of R (LinearMap.ker S.g.hom) ⟶ S.module
 
 /-- The explicit left homology data of a short complex of modules that is
 given by a kernel and a quotient given by the `LinearMap` API. -/
-@[simps K H i π]
+@[simps! i_hom π_hom, simps! -isSimp K H]
 def moduleCatLeftHomologyData : S.LeftHomologyData where
```

Comment:
If the intention is to unfold less, maybe remove the abbrevs above and make them inline in this def to avoid people from using them? Even better is to add a sentence to the docstring of this saying something like "the projections to `K` and `H` are not simp lemmas because the generic lemmas about `LeftHomologyData` is more useful here" (or any other reason for the change).

Reply:
Done, thank you. Also, I only unsimped some commuting square lemmas because they're now provable by simp. Maybe I should make them `simp high` instead?